pkg/apicast/apicast.go

```diff
@@ -67,9 +71,21 @@ func (a *APIcast) deploymentVolumeMounts() []v1.VolumeMount {
 		})
 	}
 
+	for _, customPolicySpec := range a.options.CustomPolicies {
+		volumeMounts = append(volumeMounts, v1.VolumeMount{
+			Name:      policyVolumeName(customPolicySpec),
+			MountPath: path.Join(CustomPoliciesMountBasePath, customPolicySpec.Name, customPolicySpec.Version),
```

Comment:
Are the directories automatically created if they not exist?

Reply:
yup

Reply:
:+1: thanks for the explanation